Description: Properly escaped minus signs.
  Properly escaped minus signs to fix lintian warnings like:
  I: fio: hyphen-used-as-minus-sign usr/share/man/man1/fio.1.gz:1320
Author: Martin Steigerwald <ms@teamix.de>
--- a/fio.1
+++ b/fio.1
@@ -1317,42 +1317,42 @@
 socket. 'hostname' is either a hostname or IP address, and 'port' is the port to
 listen to (only valid for TCP/IP, not a local socket). Some examples:
 
-1) fio --server
+1) fio \-\-server
 
    Start a fio server, listening on all interfaces on the default port (8765).
 
-2) fio --server=ip:hostname,4444
+2) fio \-\-server=ip:hostname,4444
 
    Start a fio server, listening on IP belonging to hostname and on port 4444.
 
-3) fio --server=ip6:::1,4444
+3) fio \-\-server=ip6:::1,4444
 
    Start a fio server, listening on IPv6 localhost ::1 and on port 4444.
 
-4) fio --server=,4444
+4) fio \-\-server=,4444
 
    Start a fio server, listening on all interfaces on port 4444.
 
-5) fio --server=1.2.3.4
+5) fio \-\-server=1.2.3.4
 
    Start a fio server, listening on IP 1.2.3.4 on the default port.
 
-6) fio --server=sock:/tmp/fio.sock
+6) fio \-\-server=sock:/tmp/fio.sock
 
    Start a fio server, listening on the local socket /tmp/fio.sock.
 
 When a server is running, you can connect to it from a client. The client
 is run with:
 
-fio --local-args --client=server --remote-args <job file(s)>
+fio \-\-local-args \-\-client=server \-\-remote-args <job file(s)>
 
-where --local-args are arguments that are local to the client where it is
-running, 'server' is the connect string, and --remote-args and <job file(s)>
+where \-\-local-args are arguments that are local to the client where it is
+running, 'server' is the connect string, and \-\-remote-args and <job file(s)>
 are sent to the server. The 'server' string follows the same format as it
 does on the server side, to allow IP/hostname/socket and port strings.
 You can connect to multiple clients as well, to do that you could run:
 
-fio --client=server2 --client=server2 <job file(s)>
+fio \-\-client=server2 \-\-client=server2 <job file(s)>
 .SH AUTHORS
 
 .B fio
